carrotServer/src/main/java/cds/carrot/org/carrotServer/service/employer/EmployerServiceImpl.java

```diff
@@ -20,18 +25,30 @@ public class EmployerServiceImpl implements EmployerService {
     private final ReviewRepository reviewRepository;
 
     @Override
-    public User getUserWithReviews(Long userId, int size) {
+    @Transactional
+    public EmployerResponseDto getUserWithReviews(Long userId, int size) {
+        if (size < 0) {
+            throw new BadRequestException(ErrorType.REQUEST_SIZE_EXCEPTION, ErrorType.REQUEST_SIZE_EXCEPTION.getMessage());
+        }
 
         UserEntity userEntity = userRepository.findById(userId)
-                .orElseThrow(() -> new RuntimeException("해당 사용자를 찾을 수 없습니다."));
+                .orElseThrow(() -> new NotFoundException(ErrorType.NOT_FOUND_USER_EXCEPTION, ErrorType.NOT_FOUND_USER_EXCEPTION.getMessage()));
 
         List<ReviewEntity> reviewEntities = reviewRepository.findByUserId(userId);
 
         int maxSize = Math.min(size, reviewEntities.size());
 
-        User user = fromUserEntityToUserMapper(userEntity, maxSize);
+        List<Review> reviews = getLimitedReviews(reviewEntities, maxSize);
 
-        return user;
+        User user = fromUserEntityToUserMapper(userEntity);
+
+        return EmployerResponseDto.of(
+                user.getUserId(),
+                user.getNickname(),
+                user.getImageUrl(),
+                user.getDegree(),
```

Comment:
java 는 kotlin 에 비해서 null 에대한 자유도가 높아서, NPE (NullPointerException) 에 예민한데요,
지금 userEntity 에 만약 degree 가 null 인 경우, NPE 가 발생할 수 있겠네요
(EmployerResponseDto 에서 degree 필드가 float, primitive type 이라 null 불가)

삼항연산자를 활용해보세요
```suggestion
                user.getDegree() == null ?  0 : user.getDegree(),
```